extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -100,11 +100,17 @@ class FlowFileRepository : public ThreadedRepository, public SwapManager {
     config_ = configure;
     std::string value;
 
-    if (configure->get(Configure::nifi_flowfile_repository_directory_default, value)) {
+    if (configure->get(Configure::nifi_flowfile_repository_directory_default, value) && !value.empty()) {
       directory_ = value;
     }
     logger_->log_debug("NiFi FlowFile Repository Directory %s", directory_);
 
+    value.clear();
+    if (configure->get(Configure::nifi_flowfile_checkpoint_directory_default, value) && !value.empty()) {
+      checkpoint_dir_ = value;
+    }
+    logger_->log_debug("NiFi FlowFile Checkpoint Directory %s", checkpoint_dir_);
```

Comment:
I think you need to either add `.string()` here, or make the conversion templates in `Logger.h` smarter to accept `path`s.

Reply:
Added fix in 9b7fd9553203ec6bfae427546e0fa214e695f17f